src/api/apis/MobyGamesAPI.ts

```diff
@@ -65,7 +66,7 @@ export class MobyGamesAPI extends APIModel {
 		console.log(`MDB | api "${this.apiName}" queried by ID`);
 
 		if (!this.plugin.settings.MobyGamesKey) {
-			throw Error(`MDB | API key for ${this.apiName} missing.`);
+			console.log(Error(`MDB | API key for ${this.apiName} missing.`));
```

Comment:
This should exit the function here.

Reply:
Oh yes that's an oversight, I was testing some things and forgot to change that one back.